planning/behavior_path_planner/src/scene_module/lane_change/lane_change_module.cpp

```diff
@@ -397,6 +413,27 @@ std::pair<bool, bool> LaneChangeModule::getSafePath(
 
 bool LaneChangeModule::isSafe() const { return status_.is_safe; }
 
+bool LaneChangeModule::isValidPath(const PathWithLaneId & path) const
+{
+  // check lane departure
+  lanelet::ConstLanelets lanes;
+  lanes.reserve(status_.current_lanes.size() + status_.lane_change_lanes.size());
+  lanes.insert(lanes.end(), status_.current_lanes.begin(), status_.current_lanes.end());
+  lanes.insert(lanes.end(), status_.lane_change_lanes.begin(), status_.lane_change_lanes.end());
+  if (lane_departure_checker_.checkPathWillLeaveLane(lanes, path)) {
```

Comment:
Due to the version problem, lanelets without extending is input to checkPathWillLeaveLane.
Is it OK?

Reply:
`getExtendedCurrentLanes` is needed, because path footprint is out of current_lanes, we need to check lane depature with lanes with next/prev lane.
![image](https://user-images.githubusercontent.com/39142679/204472484-f2ae5175-be6d-49a4-b3d2-46b55756df1e.png)

Reply:
it seems ok to use extendLanes simply like
```
lanes = util::extendLanes(lanes);
```

Reply:
Thanks! I fixed it!